.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: python
-python:
-  - "2.7"
-install: "python setup.py install"
-script: "py.test tests/unit"
 branches:
     only:
         - master
         - dogfood
+services:
+  - postgres
+python:
+  - "2.7"
+env:
+  - DB=sqlite
+  - DB=postgres
+install:
+  - sh -c "if [ $DB = postgres ]; then pip install psycopg2; fi"
+  - python setup.py install
+before_script:
+  - sh -c "if [ $DB = postgres ]; then psql -c 'CREATE DATABASE haas_travis;' -U postgres; fi"
+  - sh -c "if [ $DB = postgres ]; then cp ci/testsuite.cfg.postgres testsuite.cfg; fi"
+script: "py.test tests/unit tests/stress.py"
```

Comment:
How difficult would it be to create/destroy a random postgres DB for every test (maybe placing a hook into fresh_database())? Perhaps we could add an option to testsuite.cfg.postgres to set `[database] random = True` or something.

On my workstation, `-n auto` cut the tests in half for sqlite, which would be valuable to a developer anxiously awaiting a green checkmark :-)